concepts/language-model.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model tries to predict the next word given the already produced history.
```

Comment:
> A language model takes text input and outputs the next word or character.

Notes:
- "history" is not always true
- "word" is not always true
- I'm tempted to use "sequence" and "token" but that'll make it less understandable to many.

Reply:
Reworded.